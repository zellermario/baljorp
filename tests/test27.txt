create SJ kezdes 0
create SJ vegzes 0
addNeighbor kezdes 1 vegzes
addNeighbor vegzes 2 kezdes

create Eszkimo e vegzes
create Iglu i vegzes
create jegesmedve m kezdes

move m 1
pass m