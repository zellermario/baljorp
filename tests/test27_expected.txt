Fields:
SJ kezdes (0) - object: x
	neighbours: [(1,vegzes)]
SJ vegzes (0) - object: x
	neighbours: [(2,kezdes)]
	
Buildings:
Iglu on vegzes

Actors:
> Eszkimo eszkimo (heat: 5) on vegzes
  Inventory: []
> Jegesmedve m (heat: 1) on kezdes
  Inventory: []